src/SFA.DAS.LevyTransferMatching.Web/Views/Pledges/Applications.cshtml

```diff
@@ -35,14 +35,25 @@
         }
 
         <div class="govuk-grid-row">
-            <div class="govuk-grid-column-full">
+            <div class="govuk-grid-column-two-thirds">
 
                 <h1 class="govuk-heading-xl">
                     Transfer pledge @Model.EncodedPledgeId
                 </h1>
+               
+            </div>
+
+            <div class="govuk-grid-column-one-third">
+
+                @if (Model.Applications.Any())
+                {
+                   @* <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible app-section-break--dashed app-section-break--heavy">*@
```

Comment:
Can we remove commented code; it's in history if we ever need it again

Reply:
sorry, my bad. I have removed it now.